earth_enterprise/src/common/khxml/khxml.cpp

```diff
@@ -274,13 +379,18 @@ CreateDOMParser(void) throw()
 khxml::DOMDocument*
 ReadDocument(khxml::DOMLSParser *parser, const std::string &filename) throw()
 {
-  DOMDocument *doc = 0;
+  //khLockGuard outer(reInitOuterLock);
+  DOMDocument* doc = nullptr;
+  //{
+  //khLockGuard inner(reInitInnerLock);
 
   try {
     // Note: parseURI doesn't handle missing files nicely...returns
     // invalid doc object. Must check file existence ourselves.
     if (khExists(filename)) {
-      doc = parser->parseURI(filename.c_str());
+        notify(NFY_NOTICE,"ReadDocument, filename: %s",
```

Comment:
Is this meant to be temporary? It looks like debugging code.

Reply:
Yes, will remove